netmiko/base_connection.py

```diff
@@ -507,9 +507,9 @@ def is_alive(self) -> bool:
                 log.debug("Sending IAC + NOP")
                 # Need to send multiple times to test connection
                 assert isinstance(self.remote_conn, telnetlib.Telnet)
-                self.remote_conn.sock.sendall(telnetlib.IAC + telnetlib.NOP)  # type: ignore
-                self.remote_conn.sock.sendall(telnetlib.IAC + telnetlib.NOP)  # type: ignore
-                self.remote_conn.sock.sendall(telnetlib.IAC + telnetlib.NOP)  # type: ignore
+                self.remote_conn.get_socket().sendall(telnetlib.IAC + telnetlib.NOP)
```

Comment:
is it possible to assign this as a variable, for example:

```
telnet_socket = self.remote_conn.get_socket()
```

And then use `sendall()` on that variable.